pkg/tbtc/signing.go

```diff
@@ -19,6 +19,10 @@ import (
 	"sort"
 )
 
+// signingAttemptMaxBlockDuration determines the maximum block duration of a
+// single signing attempt.
+const signingAttemptMaxBlockDuration = 100
```

Comment:
Just flagging that this value requires more thinking. 100 blocks assuming 12sec for a single block is 20 minutes.

With 2 malicious members in a signing group, we need 5 retries of the protocol in the worst case (`P = (98 choose 51) / (100 choose 51) = 0.23757575757`), which takes 1h 40min.

With 3 malicious members in a signing group, we need 10 retries of the protocol in the worst case (`P = (97 choose 51) / (100 choose 51) = 0.11393939393`), which takes 3h 20 min.

Say we want to sweep no more than 10 deposits at once, this will take almost 17 hours for 2 malicious members case and twice that much for 3 malicious members case. Not blocking this PR, rather something for https://github.com/keep-network/tbtc-v2/pull/414/

Reply:
...plus the blocks for announcement and delay!